scripts/linking.ts

```diff
@@ -0,0 +1,74 @@
+import os from 'os';
+import path from 'path';
+import fs from 'fs-extra';
+import { execSync } from 'child_process';
+import chalk from 'chalk';
+const { promptUser } = require('../lib/prompts/promptUtils');
+
+function getSymLinksInDirectory(directory: string): string[] {
+  return fs
+    .readdirSync(directory, { recursive: false })
+    .map((item: string | Buffer<ArrayBufferLike>) => item.toString())
+    .filter((item: string) => {
+      const stats = fs.lstatSync(path.join(directory, item.toString()));
+      return stats.isSymbolicLink() && !item.includes('.bin');
+    });
+}
+
+function addPackagePrefix(packages: string[]) {
+  return packages.map((item: string) => {
+    return `${hubspotPackagePrefix}/${item}`;
+  });
+}
+
+const hubspotPackagePrefix = '@hubspot';
+const node_modules = path.join(
+  __dirname,
+  '..',
+  'node_modules',
+  hubspotPackagePrefix
+);
+const yarnLinkPath = path.join(
+  os.homedir(),
+  '.config',
+  'yarn',
+  'link',
+  hubspotPackagePrefix
+);
```

Comment:
This is where `yarn` stores the packages that are available for linking.  I.E packages you have ran `yarn link` inside of.